themes/default/content/blog/review-stacks/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Review Stacks: Collaborate in the Cloud"
+date: 2023-06-14
+meta_desc: Review Stacks give you an automated cloud environment for every pull request so your team can ship faster, safer, and more collaboratively.
+meta_image: meta.png
+authors:
+    - evan-boyle
+    - pat-gavlin
+tags:
+    - cloud engineering
+    - pulumi deployments
+    - automation api
+    - deployments
+    - review stacks
+---
+
+Today we’re excited to announce Review Stacks -- dedicated cloud environments that get created automatically every time a pull request is opened, all powered by Pulumi Deployments. Open a pull request, and Pulumi Deployments will stand up a stack with your changes and the Pulumi GitHub App will add a PR comment with the outputs from your deployment. It has never been simpler to pick up an unfamiliar codebase, make changes to **both** application *and* infrastructure code, and share a live environment for review with your teammates.
```

Comment:
```suggestion
Today we’re excited to announce Review Stacks -- dedicated cloud environments that get created automatically every time a pull request is opened, all powered by Pulumi Deployments. Open a pull request, and Pulumi Deployments will stand up a stack with your changes and the Pulumi GitHub App will add a PR comment with the outputs from your deployment. Merge the PR and Pulumi Deployments will destroy the stack and free up the associated resources. It has never been simpler to pick up an unfamiliar codebase, make changes to **both** application *and* infrastructure code, and share a live environment for review with your teammates.
```

Reply:
Do we want to say something that notes that this sort of feature has been popular for application code in things like Heroku Review Apps, but that we are now brining it to cloud infrastructure as well as applications?

Making the connection might remind people why they love this experience (and have quite likely experienced it in some other more limited context in the past)?  Also incidentally help explain the name?

Reply:
Added this elsewhere, and will continue to iterate.  I'd like to avoid direct comparison.